### ISSUE TITLE
Option to turn the script on and off

### ISSUE THREAD

**Issue by aaa1386** [euu2021/Freeplane_UtilityPanels#18]
Advantages
1- Ability to use two views of the map and keep the script active in only one view - having the script open in one of the two views opened for a map can have many benefits
2- When multiple views are open, Khalil sometimes the script does not work well. Using the suggested option, it may be possible to prevent this malfunction
3- No need to close Freeplane to end the script
4- Etc.

**Comment by euu2021** [comment]
version 1.38: Hotkey toggles panels visibility. By default, Ctrl+U (can be configured in user settings).

**Comment by euu2021** [comment]
> > version 1.38: Hotkey toggles panels visibility. By default, Ctrl+U (can be configured in user settings).
> 
>  vvv.mp4

Yes, if the "Update Selection" is activated, then the inspectors will still show. To make they hide, too, deactivate the "Update Selection". In the video, the "Update Section" button is not appearing in the first panel, because the FP screen is too small (I will fix this bug in a future version).

**Comment by euu2021** [comment]
But, I will create an option to hide everything, because I see that it can be annoying to have to toggle the Update Selection every time.

**Comment by euu2021** [comment]
1.39: option to also hide the inspectors, even if "Update Selection" is activated. This option is active, by default.

Now, there is a Settings option when right clicking a list item:

![Image](https://github.com/user-attachments/assets/3591c2e2-ce6d-4146-817d-88d5a6898132)

![Image](https://github.com/user-attachments/assets/f1316469-0633-478f-9dab-6a547bf63e2f)

The settings are saved when updating to a newer version of the script.
